src/redux/books/bookSlice.jsx

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  booksArr: [],
+  categoriesArr: [],
+};
+
+// Includes an array of books
+
+const bookSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: () => {
+    },
+    removeBook: () => {
+    },
+  },
+});
+
+export default bookSlice;
```

Comment:
- [ ] Also, please consider exporting default your `bookSlice.reducer` and export your `addBook` and `removeBook` actions! 🤓 🔧